docs/node/run-your-node/advanced/sync-node-using-state-sync.md

```diff
@@ -126,49 +126,24 @@ fetch and compare data from multiple sources.
 
 :::
 
+#### Oasis CLI
+
+If you have Oasis CLI conected to an existing node that you trust, or if your
+trust assumptions are fine with using (default) public grpc endpoints (e.g. testnet)
+run CLI's [trust command][oasis-network-trust].
```

Comment:
Repeating. Since CLI's command returns all things at once, you should mention how to obtain this data in its own (sub)section and not under very section.